include/nobleo_socketcan_bridge/socketcan_bridge.hpp

```diff
@@ -21,6 +21,14 @@ class Clock;
 
 namespace nobleo_socketcan_bridge
 {
+enum class CAN_STATE : uint8_t
```

Comment:
Why subclass of `uint8_t` if you don't specify the values? 
It might as well be a 'normal' enum then.